server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestCancelTasksAction.java

```diff
@@ -38,8 +38,10 @@ public String getName() {
 
     @Override
     public List<Route> routes() {
-        return List.of(new Route(POST, "/_tasks/_cancel"),
-            new Route(POST, "/_tasks/{task_id}/_cancel"));
+        return List.of(
```

Comment:
for the backport, you will want to be aware of https://github.com/elastic/elasticsearch/pull/54183 (if you are not already)

Reply:
👍, I've pregamed the backport locally, and that's a little annoying but not a show stopper. Just a bit more tedious manual work.